### PR TITLE
Proximity explanation

### LINKED ISSUE #134: Review HB:  clarify what proximity is, and why it is important
> (3) Overview of Protocol

    > It is not immediately clear what 'proximity' means here (the term is not listed
    > in the Terminology section). A quick recap of roles and interactions would not
    > hurt, I think, but that can also be addressed by adding more contextual text to
    > the Intro. "Most Pledges using these constrained": maybe just remove the
    > "these" - it made me read it as a back-reference to time-based vouchers. The
    > term "pinned" used here could benefit a from a tad bit more expositional text -
    > it is a term relevant to a lot of security aspects in this memo and deserves a
    > proper introduction.

### PR DESCRIPTION
**Pull request by mcr**
add explanation of proximity, remove errant word "these", and recap RFC8995 protocol.
close #134

### CODE REVIEW

File: constrained-voucher.mkd [hunk 1]
```diff
@@ -155,12 +155,20 @@ The term Registrar Voucher Request, or acronym RVR, is introduced to refer to th
 
 {{RFC8366}} provides for vouchers that assert proximity, that authenticate the Registrar and that can offer varying levels of anti-replay protection.
 
+The proximity proof provided for in {{RFC8366}}, is an assertion that the Pledge and the Registrar are believed to be in close together, from a network topology point of view.
+Like in {{RFC8955}}, proximity shown by making TLS connections between the Pledge and Registrar using IPv6 Link-Local addresses.
```
Comment (EskoDijk) [inline constrained-voucher.mkd:159]: TLS -> DTLS in our case
link-local -> that's not the case; the Join-Proxy is typically used which can relay the DTLS records to a destination an arbitrary number of hops away.

File: constrained-voucher.mkd [hunk 2]
```diff
@@ -155,12 +155,20 @@ The term Registrar Voucher Request, or acronym RVR, is introduced to refer to th
...
+
+The TLS connection is used to make a Voucher Request.
```
Comment (EskoDijk) [inline constrained-voucher.mkd:161]: TLS -> DTLS

File: constrained-voucher.mkd [hunk 3]
```diff
@@ -155,12 +155,20 @@ The term Registrar Voucher Request, or acronym RVR, is introduced to refer to th
...
+This request is verified by the an agent of the Pledge's manufacturer, which then issues a voucher.
+The voucher provides an authorization statement from the manufacturer indicating that the Registrar is the intended owner of the device.
+The voucher refers to the Registrar through pinning of the Registrar's key.
```
Comment (EskoDijk) [inline constrained-voucher.mkd:164]: This reads a bit too specific: it may pin also the Registrar's cert, or the Domain CA cert.
key -> public key ?

**Review by EskoDijk — CHANGES_REQUESTED** [review]
Some comments provided

File: constrained-voucher.mkd (hunk 1 above)
Comment (mcr) [inline constrained-voucher.mkd:159]:
```suggestion
Like in {{RFC8955}}, proximity shown by making (D)TLS connections between the Pledge and Registrar using IPv6 Link-Local addresses.
```

File: constrained-voucher.mkd (hunk 2 above)
Comment (mcr) [inline constrained-voucher.mkd:161]:
```suggestion
The (D)TLS connection is used to make a Voucher Request.
```

Comment (mcr) [inline constrained-voucher.mkd:161]:
```suggestion
The (D)TLS connection is used to make a Voucher Request.
```

File: constrained-voucher.mkd (hunk 3 above)
Comment (mcr) [inline constrained-voucher.mkd:164]:
```suggestion
The voucher refers to the Registrar through pinning of the Registrar's identity.
```

**Pull request merged**